tests/game.test.js

```diff
@@ -0,0 +1,109 @@
+const each = require('jest-each').default;
+const { Game } = require('../server/game');
+
+describe('hasTurnReset', () => {
+  each([
+    [4, 4, 4],
+    [32, 100, 100],
+    [71, 8, 5],
+    [15, 6, 7],
+  ]).it('testing turn count resets', (actualTurn, nr, nc) => {
```

Comment:
It's also usual to fill in the individual test description to follow on from the word 'it'
```suggestion
  ]).it('should reset the turn count', (actualTurn, nr, nc) => {
```
Together with the previous comment, it makes the test output read better. You then end up with an output like:
`resetGame should reset the turn count`